frontend/public/components/utils/alerts.tsx

```diff
@@ -0,0 +1,33 @@
+import * as React from 'react';
+
+export const ExpandableAlert: React.FC<CustomAlertProps> = ({alerts, type}) => {
+  const errorCount = alerts.length;
+  const [visible, setVisible] = React.useState(true);
+  const [expanded, setExpanded] = React.useState(errorCount > 1 ? false : true);
+
+  if (visible) {
```

Comment:
nit also:
```JavaScript 
return visible 
  ? <div className={`alert alert-${type}`}> ..... </div>
  : null
```
seems to be more what I see glancing around these days, but not a big deal.